mainapp/templates/mainapp/request_list.html

```diff
@@ -32,7 +32,7 @@
             {% endif %}
           </td>
           <td>{{ req.requestee }}</td>
-          <td>{{ req.requestee_phone }}</td> 
+          <td><a href="tel:{{ req.phone }}">{{ req.requestee_phone }}</a></td> 
           <td>{{ req.summarise | linebreaks }}</td>
```

Comment:
Should this be `<a href="tel:{{ req.requestee_phone }}">{{ req.requestee_phone }}</a>` instead?

Reply:
Thanks for pointing out.

Reply:
Fixed.